examples/search-index/index.json

```diff
@@ -0,0 +1,30 @@
+{
+  "AWSTemplateFormatVersion": "2010-09-09",
```

Comment:
Change file name to searchIndex.json and correct any other dependency updates that may cause